libnavigation-base/src/main/java/com/mapbox/navigation/base/logger/Logger.kt

```diff
@@ -9,7 +9,7 @@ interface Logger {
      * @param msg The message you would like logged.
      * @param tr An exception to log
      */
-    fun v(tag: String, msg: String, tr: Throwable? = null)
+    fun v(msg: String, tag: String? = null, tr: Throwable? = null)
```

Comment:
I personally prefer `tag`, `message`, `throwable` (as it was before). In any case, if we pick this order we need to fix Javadoc order too.

Reply:
I agree, typically the `tag` is the first argument.